plutus-scb/plutus-scb.yaml

```diff
@@ -4,6 +4,7 @@ dbConfig:
 
 scbWebserverConfig:
   baseUrl: http://localhost:8080
+  staticDir: plutus-scb-client/dist
```

Comment:
Should this have changed?

Reply:
This is present in `plutus-scb/plutus-scb.yaml.sample` but not here. The real solution is probably to delete `plutus-scb.yaml` and add it to gitignore...

Reply:
.. which I just did :-)